csfieldguide/chapters/content/en/big-data/sections/the-whole-story.md

```diff
@@ -0,0 +1,10 @@
+# The Whole Story
+
+Big Data gives us an opportunity to gain a much deeper understanding of how many things in the world around us work. It is a new and exciting field, and even with the huge amounts of work and research happening at the moment we are barely scratching the surface of the possibilities. It has very close ties with Artificial Intelligence, as many AI systems use Big Data, and a lot of Big Data analysis is done using AI algorithms.
```

Comment:
Maybe link to the AI chapter?

Reply:
I think I'll add that in once the new version of AI has been released, the AI content at the moment isn't super relevant I think.